app/stores/BaseStore.ts

```diff
@@ -112,7 +112,10 @@ export default abstract class BaseStore<T extends BaseModel> {
     return this.update(params, rest);
   }
 
-  get(id: string): T | undefined {
+  get(id?: string | null): T | undefined {
```

Comment:
Not loving this change…

Reply:
Agree, but the alternative is making changes at ten different places checking for `id` before calling this `get` :grimacing: